src/lua/protobuf.lua

```diff
@@ -32,13 +32,22 @@ local MAX_SINT32_LL = 2LL^31 - 1
 local MAX_SINT32_ULL = 2ULL^31 - 1
 local MAX_SINT64_ULL = 2ULL^63 - 1
 
+-- Acceptable protobuf types
+local WIRE_TYPE_VARINT = 0
+local WIRE_TYPE_I64 = 1
+local WIRE_TYPE_LEN = 2
+-- SGROUP (3) and EGROUP (4) are deprecated in proto3.
+local WIRE_TYPE_I32 = 5
```

Comment:
Can we define there constants in the wireformat module? It seems as a more appropriate place. We can use it from this module anyway, but this way the numbers are incapsulated in the low-level module.

Reply:
These constants were already defined in `protobuf_wireformat.lua` but as I found out later these constants may be contained in one structure similar to `scalars` from `protobuf.lua`